app/handlers/README.md

```diff
@@ -0,0 +1,21 @@
+This directory contains resource files e.g., HTML templates and images.
```

Comment:
This looks like README of somewhere else. Shouldn't be copied here.

Reply:
Have moved the README file to correct path under resources